src/bend.ts

```diff
@@ -137,20 +124,20 @@ export class Bend extends Modifier {
     return this;
   }
 
-  /** Get text provided in the constructor. */
-  getText(): string {
-    return this.text;
-  }
   getTextHeight(): number {
-    const textFormatter = TextFormatter.create(this.textFont);
-    return textFormatter.maxHeight;
+    const element = new Element(Category.Bend);
+    element.setText(this.phrase[0].text);
```

Comment:
It looks like the first BendPhrase is required to have a .text property.

Should .text be an optional property? In one of the test cases, I see you provided a .text with an empty string ''. Maybe it should be optional?